button-examples/Web/web-button-example.md

```diff
@@ -593,7 +595,7 @@ API and source code:
 
 The following example shows text, outlined and contained button types with Material Theming.
 
-!["Button theming example for <platform> with three buttons - text, outlined and contained - with green/black color theming and cut corners."](assets/button-theming.svg)
+!["Button theming example for <platform> with three buttons - text, outlined and contained - with green/black color theming."](assets/button-theming.svg)
```

Comment:
If Web doesn't support cut corners, then we can skip it. I was modeling the example on the Android button theme example.

In the meantime, is it possible to provide code and a screenshot of what's described in the collapsible section below?